<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HFrullatori</class>
 <widget class="QWidget" name="HFrullatori">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impostazioni frullata</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Resources/App-ark-2-icon.png</normaloff>:/Resources/App-ark-2-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbTitle">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frullatore:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="deData">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lePompe"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vasche:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="leAmount"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Linea:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leAvanzi"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Operatore:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leOperator"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leBlender"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pompe:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Quantit√†</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leLine"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leVasche"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Avanzi:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbIcon">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/Resources/verde.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Stato:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="stateSlider">
         <property name="maximum">
          <number>2</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbExport">
     <property name="text">
      <string>Esportato</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_9">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Note:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="teNote">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbInit">
       <property name="text">
        <string>Operazioni</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/Accept64.png</normaloff>:/Resources/Accept64.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrint">
       <property name="text">
        <string>Stampa</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/print-icon.png</normaloff>:/Resources/print-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/save-icon.png</normaloff>:/Resources/save-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Resources/Actions-window-close-icon.png</normaloff>:/Resources/Actions-window-close-icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>deData</tabstop>
  <tabstop>leLine</tabstop>
  <tabstop>leOperator</tabstop>
  <tabstop>leBlender</tabstop>
  <tabstop>lePompe</tabstop>
  <tabstop>leVasche</tabstop>
  <tabstop>leAvanzi</tabstop>
  <tabstop>cbExport</tabstop>
  <tabstop>teNote</tabstop>
  <tabstop>pbInit</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
